internal/resources/resource_host.go

```diff
@@ -659,9 +676,11 @@ func getVAsForHost(hostID string, vas []rest.VolumeAttachment) []rest.VolumeInfo
 			vi.TargetIQN = i.VolumeTargetIQN
 			hostvas = append(hostvas, vi)
 		}
+
+		protocol = string(i.AttachProtocol)
```

Comment:
agree.

Reply:
Thanks @mchuang3, I will revert this change. 
 @shi98382, last time we discussed about this change and the requirement of protocol field to make a decision in between filling iSCSI config or WWPNs in host. It might be possible that I was not able to clearly describe the requirement of protocol in host resource. I will revert the protocol check and update the PR.